examples/validation-app/src/controllers/coffee-shop.controller.ts

```diff
@@ -28,7 +28,7 @@ import {ValidatePhoneNumInterceptor} from '../interceptors';
 import {CoffeeShop} from '../models';
 import {CoffeeShopRepository} from '../repositories';
 
-// Add this line to apply interceptor to this class
+// Add this line to apply interceptor to this method
```

Comment:
You apply the decorator at class level and use `if` logic in the implementation to skip other methods.

Reply:
I meant to say the comment should be `// Add this line to apply interceptor to this class`